### PR TITLE
COM-311 Add free text field precise location to address hierarchy

### PR DESCRIPTION
**Pull request by rubailly**
Removed commas, quotes and trailing pipes from address field entries.

### CODE REVIEW

**Comment by prashantsable1991** [comment]
@rubailly you did a really great job but unfortunately, above "address-hierarchy-entries.csv" is not working.
 
Please check it once again by vagrant destroy.

Following warning is getting:
WARN - AddressConfigurationLoader.isMatchableLevelConfig(160) |2019-06-17 10:53:22,620| The address field 'COUNTRY' provided by the configuration doesn't match any existing address level.
WARN - AddressConfigurationLoader.loadAddressConfiguration(88) |2019-06-17 10:53:22,620| The address hierarchy configuration was not loaded because of a mismatch between the existing and provided address hierarchy levels.

Please see below openmrs.log.

[openmrs.log](https://github.com/jembi/cameroon-bahmni-metadata/files/3297004/openmrs.log)

Thanks.

**Comment by rubailly** [comment]
@prashantsable1991 The issue isn't the file, it might actually be in the way the address hierarchy module handles updating existing entries. Try cleaning up the address hierarchy module tables (...on your vagrant ) 

```
delete from address_hierarchy_address_to_entry_map;
delete from address_hierarchy_entry;
delete from address_hierarchy_level;
```

...and restart the openrms service. The entries should be loaded without errors.

We might have to create a separate story to handle this issue, I don't think it is in the scope of the original story.

**Comment by lauravignoli** [comment]
@rubailly We need to finish this story now. If running that sql is what's needed we can add a sql file to the sql directory so it's run in dev and staging. Please test and do that. Thanks :)

**Comment by prashantsable1991** [comment]
@rubailly I agreed to @lauravignoli comment above.

**Comment by rubailly** [comment]
@prashantsable1991 @lauravignoli ...let me know if it works for you this time.

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lauravignoli) [inline .project]: @rubailly please delete this file

**Pull request merged**